backend/icpy/services/whatsapp/whatsapp_bot_service.py

```
@@ -792,23 +955,66 @@ async def _send_formatted_response(self, chat_jid: str, full_response: str):
+        try:
+            abs_path = os.path.abspath(media_path)
+            roots = []
+            workspace_root = getattr(self.chat_service, "workspace_root", None)
+            if workspace_root:
+                roots.append(os.path.abspath(workspace_root))
+            # Current working directory as a fallback root
+            roots.append(os.path.abspath(os.getcwd()))
+            return any(
+                os.path.commonpath([abs_path, root]) == root for root in roots
+            )
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)